server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -323,26 +309,33 @@ public void testCreateTableWithIndex() throws Exception {
     @Test
     @UseNewCluster
     public void testCreateTableWithCompositeIndex() throws Exception {
-        execute("create table novels (title string, description string, " +
-                "index title_desc_fulltext using fulltext(title, description) " +
-                "with(analyzer='stop')) with (number_of_replicas = 0)");
-
-        Map<String, Object> expectedMapping = Map.of(
-            "dynamic", "strict",
-            "_meta", Map.of("indices", Map.of("3", Map.of())),
-            "properties", Map.of(
-                "title", Map.of("type", "keyword", "position", 1, "oid", 1),
-                "description", Map.of("type", "keyword", "position", 2, "oid", 2),
-                "title_desc_fulltext", Map.of(
-                    "type", "text",
-                    "position", 3,
-                    "oid", 3,
-                    "analyzer", "stop",
-                    "sources", List.of("1", "2")
+        execute("""
+            create table novels (
+                title string,
+                description string,
+                index title_desc_fulltext using fulltext(title, description) with (analyzer='stop')
+            ) with (number_of_replicas = 0)
+            """
+        );
+        DocTableInfo table = getTable("novels");
+        assertThat(table.indexColumns()).satisfiesExactly(
+            x -> assertThat(x)
+                .hasName("title_desc_fulltext")
+                .hasType(DataTypes.STRING)
+                .hasAnalyzer("stop")
+                .hasSourceColumnsSatisfying(
+                    s -> assertThat(s).hasName("title"),
+                    s -> assertThat(s).hasName("description")
```

Comment:
Don't know if it's important but we don't assert now:
- oid
- position
- sources

Reply:
The source is just a link to the actual column so it shouldn't be important here.